### PR TITLE
Feature/cvsb 14484

### PR DESCRIPTION
**Pull request by JonLazariniDD**
add font as part of the static build

https://jira.dvsacloud.uk/browse/CVSB-14484

### CODE REVIEW

File: src/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+}
+/* latin */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
+  src: local('Roboto Medium'), local('Roboto-Medium'),
+    url(assets/font/KFOlCnqEu92Fr1MmEU9fBBc4AMP6lQ1.woff2) format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
```
Comment (alleypa) [inline src/_fonts.scss:189]: needs extra line

File: src/_icons.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  line-height: 1;
+  letter-spacing: normal;
+  text-transform: none;
+  display: inline-block;
+  white-space: nowrap;
+  word-wrap: normal;
+  direction: ltr;
+  -webkit-font-feature-settings: 'liga';
+  -webkit-font-smoothing: antialiased;
+}
```
Comment (alleypa) [inline src/_icons.scss:23]: needs extra line

File: src/index.html [hunk 3]
```diff
@@ -26,15 +18,6 @@
     <!-- add to homescreen for ios -->
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="apple-mobile-web-app-status-bar-style" content="black" />
```
Comment (m-ujaffer) [inline src/index.html:20]: I think we can remove these meta tags as well as they were needed for angular ionic.

**Comment by alleypa** [comment]
also lets squash the commits. approval is pending

File: src/index.html (hunk 3 above)
Comment (JonLazariniDD) [reply src/index.html:20]: thanks for spotting this 👍
done (squashing and rebasing)

**Comment by JonLazariniDD** [comment]
build: https://jenkins.cvs.dvsacloud.uk/job/UPDATE__BRANCH/job/job_feature_deploy_vtm_14484/5/console

**Pull request merged**